tests/roles/stop_remaining_services/defaults/main.yaml

```diff
@@ -1,3 +1,5 @@
 edpm_node_hostname: standalone.localdomain
 install_yamls_path: /home/zuul/src/github.com/openstack-k8s-operators/install_yamls/
 edpm_privatekey_path: "{{ install_yamls_path }}/out/edpm/ansibleee-ssh-key-id_rsa"
+edpm_computes: |-
+  ["{{ edpm_node_hostname }}"]="{{ edpm_node_ip }}"
```

Comment:
didn't get far enough to test this bit in last run so will re test today (hitting https://issues.redhat.com/browse/OSPCIX-320)

Reply:
last run green 

https://logserver.rdoproject.org/32/52732/8/check/periodic-data-plane-adoption-multinode-osp-17-to-extracted-crc-no-ceph/08d734f/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_minimal_out_2024-06-06T12:30:57EDT.log